datalad/distributed/ora_remote.py

```diff
@@ -1172,11 +1172,11 @@ def whereis(self, key):
         if isinstance(self.io, HTTPRemoteIO):
             # display the URL for a request
             # TODO: method of HTTPRemoteIO
-            return self.ria_store_url[4:] + "/annex/objects" + \
-                   self.annex.dirhash(key) + "/" + key + "/" + key
+            return self.ria_store_url[4:] + "/annex/objects" + "/" + \
```

Comment:
Non-critical nitpick: Why `/annex/objects" + "/"` instead of `"/annex/objects/"? ;-)

Reply:
> Non-critical nitpick: Why /annex/objects" + "/" instead of "/annex/objects/"? ;-)

Was mirroring your approach, in an attempt to be consistent :)

Reply:
:)

Yes, but I was adding to a variable. You are adding two constants. That's unnecessary.

Reply:
ok, changed it